scripts/test

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh
-# Run all tests
-#
+# Summary: Run all tests. Requires that you have an es instance running locally
+#   or are pointing to one with an env var ELASTICSEARCH_HOST.
 
-py.test "$@"
+(
+  # use parens here to prevent exporting variable to parent env
+  if [ -z "$ELASTICSEARCH_HOST" ]; then
+    # default to localhost if this var isn't already set
+    export ELASTICSEARCH_HOST=localhost
+  fi
```

Comment:
Yeah, but you don't even have to add this line. The code already checks for a "ELASTICSEARCH_HOST" environment variable, else defaults to "localhost".

https://github.com/theonion/django-bulbs/blob/master/example/settings.py#L103

Just add this to your `~/.bashrc`, `~/.bash_profile`, etc.:

`export ELASTICSEARCH_HOST=dockerhost`